Dfe.Academies.Academisation.Service.UnitTest/Commands/Legacy/Project/EnrichProjectCommandTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoFixture;
+using Dfe.Academies.Academisation.IData.Establishment;
+using Dfe.Academies.Academisation.IData.ProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
+using Dfe.Academies.Academisation.Service.Commands.Legacy.Project;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Dfe.Academies.Academisation.Service.UnitTest.Commands.Legacy.Project
+{
+	public class EnrichProjectCommandTests
+	{
+		private readonly EnrichProjectCommand _sut;
```

Comment:
What does _sut stand for?

Reply:
system under test
Can change to subject if that's clearer?

Reply:
No need, now I know the above it makes sense.